pkg/commands/drift/flags/driftissue.go

```diff
@@ -54,7 +50,7 @@ func (d *DriftIssueFlags) Register(set *cli.FlagSet, options *Options) {
 		Target:  &d.FlagGitHubIssueLabels,
 		Example: "guardian-iam-drift",
 		Usage:   `The labels to use on any created GitHub Issues.`,
-		Default: []string{options.DefaultIssueLabel},
+		Default: []string{},
```

Comment:
Should this be removed and do a nil check instead? What if someone wants zero labels to be given?

Reply:
That would be problematic as we manage the issues based on these labels (lookup, close, etc). I wouldn't want to add that complexity to this tool